src/stats.rs

```diff
@@ -125,6 +125,15 @@ impl GooseStats {
             } else {
                 0.0
             };
+            let req_s;
+            let fail_s;
+            if self.duration == 0 {
+                req_s = 0.to_formatted_string(&Locale::en);
+                fail_s = 0.to_formatted_string(&Locale::en);
+            } else {
+                req_s = (total_count / self.duration).to_formatted_string(&Locale::en);
+                fail_s = (request.fail_count / self.duration).to_formatted_string(&Locale::en);
+            }
```

Comment:
This belongs into a helper function - if you don't want to make it private it can be a function within the function - taking `duration, total_count, fail_count` as parameters - please keep code DRY ;) (Don't repeat yourself)

Reply:
Duplicated code moved into a helper function, with a test.